Changelogs for Digital Twin Manager Plugins 2.0.0 (#2834)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-0-0-support-for-default-asset-model-for-groups.md

---
date: ""

> title: "Support for default asset model for groups"

Reviewer: ```suggestion
title: "Added support in DTM plugins for default asset model for groups"
```

> title: "Added support in Dtm-plugins for default asset model for groups"

Reviewer 2: ```suggestion
title: "Added support in DTM plugins for default asset model for groups"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1437"
version: "1021.0.0"
---

> The DTM plugins now support creation and viewing of groups created using the default asset model for groups.

Reviewer 2: ```suggestion
The DTM plugins now support creating groups and viewing groups created using the default asset model for groups.
```

Reviewer 2: Here it says "DTM plugins". In other cases I see "dtm-plugin". Is this something different? If not, should be unified.

Reviewer 3: @BeateRixen , the name of the plugin app is Dtm-plugins and the plugins are collectively called DTM plugins.

> The DTM plugins now support creating groups and viewing groups created using the default asset model for groups.

Reviewer: ```suggestion
DTM plugins now support creating and visualizing groups based on their default asset model. Users can create multiple groups simultaneously and organize devices within these groups. The updated view also displays the current labels and icons assigned to the default asset model for groups.
```

> Dtm-plugins now support creating and visualizing groups based on their default asset model. Users can create multiple groups simultaneously and organize devices within these groups. The updated view also displays the current labels and icons assigned to the default asset model for groups.

Reviewer 2: ```suggestion
DTM plugins now support creating and visualizing groups based on their default asset model. Users can create multiple groups simultaneously and organize devices within these groups. The updated view also displays the current labels and icons assigned to the default asset model for groups.
```
